lib/srv/desktop/tdp/proto.go

```diff
@@ -1469,8 +1467,10 @@ const tdpMaxNotificationMessageLength = 10240
 // The limit is kept as an additional defense-in-depth measure.
 const tdpMaxPathLength = 10240
 
-const maxClipboardDataLength = 1024 * 1024    // 1MB
-const tdpMaxFileReadWriteLength = 1024 * 1024 // 1MB
+const (
+	maxClipboardDataLength    = 1024 * 1024 // 1MB
+	tdpMaxFileReadWriteLength = 1024 * 1024 // 1MB
+)
```

Comment:
Guessing this was an auto-generated change?

We have several consts here, and two of them were combined. I'd revert the change or put all of the consts together.  Grouping some but not all of them makes it seems like these 2 values are in some way related, and I don't think that's actually the case.

Reply:
Yep that was a change from a formatter, I'll group all the consts in the same block